### PR TITLE
Sign Up Welcome Message: UI

### PR DESCRIPTION
**Pull request by diVladimirov**
изображение_2022-08-16_221050207

### CODE REVIEW

File: src/components/Sample/Sample.tsx [hunk 1]
```diff
@@ -2,9 +2,10 @@ import React, { FC } from 'react';
 import { useAppSelector, useAppDispatch } from 'redux/hooks';
 import { increment, decrement } from 'redux/reducers/sampleSlice';
 import type { RootState } from 'redux/store';
-import { TitleStyled } from 'Sample.styles';
+import { TitleStyled } from './Sample.styles';
 import { useTranslation } from 'react-i18next';
-import GoogleAuth from 'GoogleAuth/GoogleAuth';
+// import GoogleAuth from 'GoogleAuth/GoogleAuth';
+import GoogleAuth from 'components/GoogleAuth/GoogleAuth';
```
Comment (lyudad) [inline src/components/Sample/Sample.tsx:8]: why do you add anything to the sample? its just exaple

File: src/components/WelcomePage/WelcomePageLayout.styles.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+
+export const WelcomePageLayoutWrapper = styled.div`
+  text-align: center;
+`;
+
+export const TitleStyled = styled.div`
+  font-size: 40px;
```
Comment (lyudad) [inline src/components/WelcomePage/WelcomePageLayout.styles.ts:8]: all fonts should be in constants

File: src/components/WelcomePage/WelcomePageLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react';
+import { Link } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { WelcomePageLayoutWrapper, TitleStyled, ImageStyled } from './WelcomePageLayout.styles';
+import ArrowDowm from '../../assets/arrowDown.jpg';
```
Comment (lyudad) [inline src/components/WelcomePage/WelcomePageLayout.tsx:5]: no, only absolute path. please check common chat

Comment (diVladimirov) [reply src/components/WelcomePage/WelcomePageLayout.tsx]: fix mistake

File: src/components/WelcomePage/WelcomePageLayout.styles.ts (hunk 2 above)
Comment (diVladimirov) [reply src/components/WelcomePage/WelcomePageLayout.styles.ts]: fix mistake

File: src/components/Sample/Sample.tsx (hunk 1 above)
Comment (diVladimirov) [reply src/components/Sample/Sample.tsx]: it was wrong path and I fix it

**Pull request merged**